docs/shared/recipes/root-level-scripts.md

```diff
@@ -0,0 +1,71 @@
+# Run Root-Level NPM Scripts with Nx
+
+There are often tasks in a codebase that apply to the whole codebase rather than a single project. Before version 15.3.0, the only way to use Nx's task pipelines or caching mechanisms for those scripts was to create an empty `scripts` project and run the scripts from there. Now you can run those commands directly from the root `package.json`.
+
+## Example
+
+Let's say your root `package.json` looks like this:
+
+```json
```

Comment:
Can we use the `json {% fileName="package.json" %}` syntax for these so it shows the filename the code belongs to? 
I know it's in the text, but it is easier to understand where this file as well as the `project.json` are when you see it directly above the code example :)